src/utils/system.js

```diff
@@ -18,6 +18,6 @@ module.exports = class System {
   }
 
   static processReadyAt () {
-    return new Date(new Date() - process.uptime() * 1000)
+    return ((new Date() - process.uptime() * 1000) / 1000).toFixed(0)
```

Comment:
This is not really related to PR, but could you change this like below?
```suggestion
    return new Date(Date.now() - process.uptime() * 1000)
```